src/common/reducers/user.js

```diff
@@ -1,4 +1,16 @@
 // initialised with server side rendered state
-export function user(state = null){
-  return state;
+
+export const DEBUG_SET_USER = 'DEBUG_SET_USER';
+
+export function user(state = null, action) {
+  switch(action.type) {
+    // set individual user properties for debugging
+    case DEBUG_SET_USER:
```

Comment:
We don't have any action creators for this (as it's only set serverside), but it was useful when debugging to be able to set/change any user info from Redux dev tools.